### PR TITLE
add-comments

### PR DESCRIPTION
**Pull request by Kobeszkie**
I add nickname in "show.html.erb"

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -5,6 +5,7 @@
       <h1>
         <%= gravatar_for @user %>
         <%= @user.name %>
+          Kobeszkie
```
Comment (moseslucas) [inline app/views/users/show.html.erb:8]: This is good but can you wrap this around a `p` element? Like this

```suggestion
          <p>Kobeszkie</p>
```

**Review by moseslucas — CHANGES_REQUESTED** [review]
Just one small suggestion

**Comment by Kobeszkie** [comment]
i follow your suggestion already!

**Review by moseslucas — APPROVED** [review]
Nice
jesusapprove

**Pull request merged**